gradle/dependencies.gradle

```diff
@@ -1,9 +1,19 @@
 ext.deps = [
     kotlin: [
-        stdlib: "org.jetbrains.kotlin:kotlin-stdlib",
-        junit: "org.jetbrains.kotlin:kotlin-test-junit",
+        stdlib: "org.jetbrains.kotlin:kotlin-stdlib-common",
+        test: [
+            annotations: "org.jetbrains.kotlin:kotlin-test-annotations-common",
+            common: "org.jetbrains.kotlin:kotlin-test-common",
+            jvm: "org.jetbrains.kotlin:kotlin-test-junit",
+            js: "org.jetbrains.kotlin:kotlin-test-js",
+        ],
     ],
-    okio: "com.squareup.okio:okio:2.6.0",
+    serialization: [
+        common: "org.jetbrains.kotlinx:kotlinx-serialization-runtime-common:0.20.0",
+        jvm: "org.jetbrains.kotlinx:kotlinx-serialization-runtime:0.20.0",
+        js: "org.jetbrains.kotlinx:kotlinx-serialization-runtime-js:0.20.0",
```

Comment:
Might be good to use the recently released `1.0.0-rc`.

Reply:
Updated the project to use the `1.0.0-RC`. This eliminated the `js` and `jvm` variations of the dependencies. Also, it did require Kotlin 1.4